dashboard/client/package.json

```diff
@@ -62,18 +66,32 @@
       "*.svg",
       "*.css"
     ],
+    "parser": "@typescript-eslint/parser",
     "extends": [
       "plugin:import/warnings",
+      "plugin:@typescript-eslint/recommended",
       "react-app"
     ],
     "plugins": [
-      "prefer-arrow"
+      "prefer-arrow",
+      "@typescript-eslint"
     ],
     "rules": {
       "@typescript-eslint/consistent-type-definitions": [
         "error",
         "type"
       ],
+      "@typescript-eslint/no-explicit-any": 0,
```

Comment:
sad sad. Are there a lot of `any` in the codebase?

Reply:
yes there are a few. I think this is fine. Sometimes, having the "any" escape hatch will be necessary when dealing with third-party libraries.

At least implicit anys are still banned, so if someone is using any, they have to very explicitly set it in the code.